packages/sources/binance/test/integration/adapter.test.ts

```diff
@@ -76,3 +92,85 @@ describe('execute', () => {
     })
   })
 })
+
+describe('websocket', () => {
+  let mockedWsServer: InstanceType<typeof MockWsServer>
+  let server: http.Server
+  let req: SuperTest<Test>
+
+  let oldEnv: NodeJS.ProcessEnv
+  beforeAll(async () => {
+    if (!process.env.RECORD) {
+      mockedWsServer = mockWebSocketServer('wss://stream.binance.com:9443/ws')
```

Comment:
Suggestion: you can pull the ws endpoint from `DEFAULT_WS_API_ENDPOINT` in `config.ts`